phone_actions.rb

```diff
@@ -0,0 +1,15 @@
+class PhoneActions
```

Comment:
It could be imported in other place, but maybe is missing the require for the `rest-client` gem
```suggestion
require 'rest-client'

class PhoneActions
```